extensions/sql-migration/src/dialog/assessmentResults/assessmentResultsDialog.ts

```diff
@@ -58,7 +58,7 @@ export class AssessmentResultsDialog {
 	public async openDialog(dialogName?: string) {
 		if (!this._isOpen) {
 			this._isOpen = true;
-			this.dialog = azdata.window.createModelViewDialog(this.title, this.title, '90%');
+			this.dialog = azdata.window.createModelViewDialog(this.title, this.title, 'wide');
```

Comment:
is this intentional?

Reply:
Yes, there was a lot of blank space and this gets rid of that.

Reply:
![image](https://user-images.githubusercontent.com/6816294/122320695-833c7500-ced7-11eb-8906-9bc3f48ea9e1.png)